draft/2021-12-01-this-week-in-rust.md

```diff
@@ -16,22 +16,50 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 ### Official
 
-### Foundation
+* [In response to the moderation team resignation](https://blog.rust-lang.org/inside-rust/2021/11/25/in-response-to-the-moderation-team-resignation.html)
+* [1.57.0 pre-release testing](https://blog.rust-lang.org/inside-rust/2021/11/30/1.57.0-prerelease.html)
 
 ### Project/Tooling Updates
 
+* [Rust Analyzer Changelog #105](https://rust-analyzer.github.io//thisweek/2021/11/29/changelog-105.html)
 * [SixtyFPS (GUI crate): Changelog for 28th of November 2021 – 0.1.15 Release](https://sixtyfps.io/thisweek/2021-11-29.html)
 
 ### Observations/Thoughts
 * [Do we Really Need Undefined Behavior?](https://www.ralfj.de/blog/2021/11/24/ub-necessary.html)
 * [Box Pointers in Rust make recursive types possible](https://blog.knoldus.com/box-pointers-in-rust-make-recursive-types-possible/)
+* [Notes On Module System](https://matklad.github.io//2021/11/27/notes-on-module-system.html)
+* [A prioritised micro-batch scheduler in rust](https://njk.onl/blog/gaffer.html)
+* [Go footguns: Go Defer and Rust Drop](https://dev.to/mark_saward/go-footguns-go-defer-and-rust-drop-17af)
+* [LLVM internals, part 4: attributes and attribute groups](https://blog.yossarian.net/2021/11/29/LLVM-internals-part-4-attributes-and-attribute-groups)
+* [Implementing Raft for Browsers with Rust and WebRTC](https://eevans.co/blog/wraft/)
+* [Hubris and Humility](https://oxide.computer/blog/hubris-and-humility)
+* [audio] [Game Development with Herbert Wolverson](https://rustacean-station.org/episode/048-herbert-wolverson/)
+* [video] [Rust Linz, November 2021 - Advent of Code by Rainer Stropek with Fred Morcos](https://www.youtube.com/watch?v=bejThbzAYOA)
+* [video] [Rust Linz, November 2021 - Serde Shenanigans by Armin Ronacher](https://www.youtube.com/watch?v=UhZGYS13twc)
+* [video] [Rust Linz, November 2021 - WebAssembly beyond the browser by Thorsten Hans](https://www.youtube.com/watch?v=o6qe4lw9-8c)
+* [video] [Rust 2021 Edition Explained](https://www.youtube.com/watch?v=uIeIBlB0JFw)
```

Comment:
the YouTube links are not shortened

Reply:
hm I can't find it anymore where we discussed that but I thought we were supposed to use this `https://youtu.be/...` notation - but now I am not sure anymore

Reply:
I thought we decided that we would replace things that redirect with whatever they redirect to. That's the way I wrote it in the README, anyway.

Reply:
ok damn then I did it the wrong way around last week, nvm